ByLaws.md

```diff
@@ -5,105 +5,171 @@
        ## TABLE OF CONTENTS ##
 
 NAME ………………..…………………………………………………………………………….1
+
 Name ………….…………………………………………………………………………………….1
 
 OFFICES………………………………………………………………………………………...1
+	
 	Principal Office……………………………………………..1
+
 Other Offices…………………………………………………………………………..1
 
 PURPOSES……………………………………………………………………………......1
+
 General Purposes……………………………………………………………....1
+
 Specific Purpose…………………………………………………………………….1
 
 SHAREHOLDING…………………………………………………………………………….1-3
+
 Authorized Number of Shares…………………………………………1
+
 Preferred Stock A……………………………………………………………………1
+
 Preferred Stock B………………………………………………………………….2
+
 Payment for Shares……………………………………………………………….2
+
 Vesting of Shares………………………………………………………………….2
+
 Dividend………………………………………………………………………………………..2
+
 Section 4.7	Pre-emptive Right & Right of First Refusal………………………………………………………2
+
 Section 4.8	Redemption of Shares……………….3
+
 Section 4.9	Transfer of Shares………………………3
+
 Section 4.10	Share Certificates…………………….3
 
 MEMBERS……..………………………………………………………………………………..4
+
 Members….……………………………………………………………………………………….4
 
 BOARD OF DIRECTORS………………………………………………………………4-5
+
 General Corporate Powers……………………………………………….4
+
 Specific Powers……………………………………………………………………….4
+
 Authorized Number ……………………………………………………………….4
+
 Election, Designation, and Term of Office….4
+
 Events Causing Vacancy…………………………………………………….5
+
 Resignation………………………………………………………………………………….5
+
 Filling Vacancies………………………………………………………………….5
 
 OFFICERS…………………………..…………………………………………………………5-8
 	Officers ………………………………………………………………….5
+
 Election of Additional Officers………………………………6
+
 Removal of Officers…………………………………………………………….6
+
 Resignation of Officers………………………………………………….6
+
 Responsibilities of Officers…………………………………….6
+
 7.5.1            Chief Executive Officer………6
-         7.5.2            President………………….6
-         7.5.3            Chief Financial Officer / Treasurer……….………………….......6
-                 7.5.3(a) Books of Account…6
-                 7.5.3(b) Financial Reports…………………………………………………………………………..………………7
-                 7.5.3(c) Deposit and Disbursement of Money and Valuables….....7
+        
+7.5.2                     President………………….6
```

Comment:
This section needs to be indented

Reply:
created another pull req

Reply:
You can just update this branch rather than creating more pull request. Just push to the same branch and it will update automatically.